subsys/net/l2/ethernet/arp.c

```diff
@@ -467,7 +467,7 @@ void net_arp_update(struct net_if *iface,
 	struct net_pkt *pkt;
```

Comment:
Looks good, minor nit about the commit subject, please update it to
`net: arp: Fix dead lock caused by arp_mutex`

Reply:
Hi jukkar, have updated the commit title, thanks